db/seeds/base/add_declarations.rb

```diff
@@ -1,106 +1,64 @@
 # frozen_string_literal: true
 
-helpers = Class.new { include ActiveSupport::Testing::TimeHelpers }.new
+if Rails.env.in? %w[development review]
```

Comment:
I don't think we need this in here - the declaration seeds don't need to be handled differently to the contract seeds for example

Reply:
Yeh, think we want to be able to tests just as well locally as in a review app and we'll need it to run in staging as well